clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookMessageHandlerTest.spec.ts

```diff
@@ -321,4 +324,114 @@ describe("Test the copybook message handler", () => {
     );
     expect(unreachable).not.toHaveBeenCalled();
   });
+
+  it("checks copybooks search respects processor group definitions order", async () => {
+    SettingsService.getCopybookExtension = jest
+      .fn()
+      .mockReturnValue(Promise.resolve([""]));
+    SettingsService.getCopybookLocalPath = jest
+      .fn()
+      .mockReturnValue(Promise.resolve([]));
+    SettingsService.getDsnPath = jest
+      .fn()
+      .mockReturnValue(["/configured/path"]);
+
+    const zoweApi: IApiRegisterClient = {
+      getExplorerExtenderApi: unreachable,
+      getUssApi: unreachable,
+      getMvsApi: unreachable,
+      registeredApiTypes: unreachable,
+    };
+
+    const downloader = new CopybookDownloadService("/storagePath", zoweApi, {
+      isEndevorElement(_uri: string) {
+        return false;
+      },
+      onDidChangeElement: unreachable,
+      listMembers,
+      listElements,
+      getMember: unreachable,
+      getElement: unreachable,
+      async getProfileInfo(_uri) {
+        return Promise.resolve({
+          instance: "instance",
+          profile: "profile",
+        });
+      },
+      getConfiguration: unreachable,
+    });
+    SettingsService.getUssPath = jest.fn().mockReturnValue(["uss/path"]);
+    SettingsService.getDsnPath = jest.fn().mockReturnValue(["dsn/path"]);
+    downloader["dsnDownloader"]!.hasMember = jest.fn().mockResolvedValue(false);
+    downloader["ussDownloader"]!.hasMember = jest.fn().mockResolvedValue(false);
+    downloader["e4eDownloader"]!.hasElement = jest.fn().mockResolvedValue(true);
```

Comment:
I think this has the same problem. To verify ordering, the member/element needs to be available from multiple sources.